### ISSUE TITLE
Add progress bars to the UI

### ISSUE THREAD

**Issue by KevinMcGee** [DigitalGlobe/GGD-OpenSpaceNet#100]
We need to take the progress bars from the current OSN processing and push them up to the UI using QT progress bars. We need to build this in a way where the UI can support N progress bars to future proof the concurrent projects coming up later down the pipeline.

Some suggestions from @avitebskiy:

Just do it for SimpleProgress for now.

your progress report should just display multiple progress bars, each with its own title.

Look at how MultiProgressReport works. Do it that way.

Basically take the MultiProgressReport that's already there, make it inherit from a generic interface, then implement that interface in QT as well
